src/main/java/net/snowflake/client/core/SessionUtilKeyPair.java

```diff
@@ -66,6 +75,14 @@ class SessionUtilKeyPair {
 
   private static final int JWT_DEFAULT_AUTH_TIMEOUT = 10;
 
+  /** provider name */
+  private static final String BOUNCY_CASTLE_PROVIDER = "BC";
+
+  /** provider name for FIPS */
+  private static final String BOUNCY_CASTLE_FIPS_PROVIDER = "BCFIPS";
+
+  private boolean ENABLE_BOUNCYCASTLE_PROVIDER = true;
```

Comment:
This is still a work in progress. I'm just testing things out to see how this will function if we default to true, but I think we're going to want it false for sure and then make it configurable.